sdcm/tester.py

```diff
@@ -2049,6 +2061,46 @@ def get_truncated_time_from_system_truncated(self, session, table_id):  # pylint
         truncated_time = self.rows_to_list(session.execute(query))
         return truncated_time[0]
 
+    def get_describecluster_info(self) -> Optional[ClusterInformation]:
+        """
+        Runs the 'nodetool describecluster' command on a node.
+
+        Example output:
+
+        nodetool describecluster
+
+        Cluster Information:
+             Name: Test Cluster
+             Snitch: org.apache.cassandra.locator.SimpleSnitch
+             Partitioner: org.apache.cassandra.dht.Murmur3Partitioner
+             Schema versions:
+                     86a67fc7-1d7c-3dc3-9be9-9c86b27e2506: [172.17.0.2]
+
+        The output is then packaged into a ClusterInformation
+        namedtuple, for easier access to the fields.
+        """
+        node = self.db_cluster.get_node()
+        describecluster_output = node.run_nodetool(sub_cmd="describecluster")
+
+        if describecluster_output.ok:
+            lines = describecluster_output.stdout.splitlines()
+            for line in lines:
+                name = line.split(":")[-1].strip() if "Name" in line else "N/A"
```

Comment:
In the end of things name/snitch/partitioner will be "N/A"
To prevent it:

```
value = line.split(":")[-1].strip()

if "Name" in line:
   name = value
elif "Snitch" in line:
   snitch = value
elif "Partitioner" in line:
   partitioner = value
else:
   if "[" in line:
       split_line = line.split(":")
    ....
```